scripts/zones/Port_Jeuno/npcs/Sagheera.lua

```diff
@@ -251,9 +251,33 @@ local abcShop =
     [20] = { item =  2127, abc =  75 }, -- metal_chip
 }
 
+-----------------------------------
+-- trades chips for ancient beastcoins
+-----------------------------------
+local tier1Chips =
```

Comment:
Since the same logic seems to happen twice for both of these tables, could combine into one with the item as key, with its chipValue as value

Reply:
That's what I initially did but I use these tables for checking the trade.
https://github.com/LandSandBoat/server/pull/2765/files#diff-0bab3dbe20460986828316156b5e730a1831f9237a333abc455b87e8a97101e5R316